packages/types/src/messages/describe-sequence.ts

```diff
@@ -27,7 +27,7 @@ export type FunctionDefinition = {
      */
     name?: string;
     /**
-     * Addtional description of the function
+     * Additional description of the function
```

Comment:
There are two more typos in this file, maybe you could fix them while you are here 😉 
- L6:  funcion -> function
- L8: neighbours -> neighbors 

thanks 💪🏼